src/screens/tabs/Tokens/index.js

```diff
@@ -23,12 +23,12 @@ import styles from './styles';
 
 function Tokens() {
   const dispatch = useAppDispatch();
+  const navigation = useNavigation();
```

Comment:
Should we use the navigation that comes from the params?

Reply:
yep